packages/web-components/src/utils/dom.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Swap the DOM elements at each index
+ */
+function swapChildren(rows: NodeList, i: number, j: number): void {
```

Comment:
Could you use some more descriptive param names? It's not clear what `i` and `j` are.